app/src/test/java/com/anbroidsdev/converter/OpenExchangeRatesJsonParserTest.java

```diff
@@ -0,0 +1,90 @@
+package com.anbroidsdev.converter;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.Currency;
+import java.util.Map;
+
+import static junit.framework.Assert.assertEquals;
+import static junit.framework.Assert.assertNotNull;
+
+/**
+ * Created by david on 9/3/15.
+ */
+public class OpenExchangeRatesJsonParserTest {
+
+    RatesJsonParser parser;
+
+    String validResponse = "{\n" +
+            "  \"disclaimer\": \"disclaimer\",\n" +
+            "  \"timestamp\": 1425916861,\n" +
+            "  \"base\": \"USD\",\n" +
+            "  \"rates\": {\n" +
+            "    \"EUR\": 0.921691,\n" +
+            "    \"GBP\": 0.66226,\n" +
+            "    \"JPY\": 121.0982\n" +
+            "  }\n" +
+            "}";
+
+    String appIdNotValidResponse = "{\n" +
+            "    \"error\" : true,\n" +
+            "    \"status\" : 400,\n" +
+            "    \"message\" : \"invalid_app_id\",\n" +
+            "    \"description\" : \"Invalid App ID - please sign up at https://openexchangerates.org/signup, or contact support@openexchangerates.org.\"\n" +
+            "}";
+
+    @Before
+    public void setUp(){
+
```

Comment:
Ok, por mí bien el git-hook. Por ahí he visto varios archivos de reglas de estilo para configurar Android studio, por ejemplo los de Android Platform
(https://raw.githubusercontent.com/android/platform_development/master/ide/eclipse/android-formatting.xml)[https://raw.githubusercontent.com/android/platform_development/master/ide/eclipse/android-formatting.xml] no se si se parecen al estilo que usáis